### ISSUE TITLE
Universal navigation links undefined

### ISSUE THREAD

**Issue by rdunk** [mplusmuseum/mplusmuseum-stories#254]
Some of the links in the universal navigation don't seem to work on staging, I believe it might because of the function linked below which seems to be used for top level links, but top level links don't seem to have an `external_link` property as far as I can see?

https://github.com/mplusmuseum/mplusmuseum-stories/blob/master/src/components/AppHeaderDropdown.vue#L155-L158

**Comment by arnasziedavicius** [comment]
@rdunk rightly observed and is an issue previously noted I believe. Please see solution in pull request #255

**Comment by chrismplus** [comment]
Hi @rdunk, @arnasziedavicius—

The issue regarding the universal nav links was previously fixed. It seems that on the new Staging Site (https://staging-stories.mplus.org.hk/en/) that these fixes have been overridden or not incorporated. 

**FE:**
![image](https://user-images.githubusercontent.com/26731347/57053789-c836ca80-6cc2-11e9-9add-5e88b786323f.png)
(note URL in bottom left hand corner)

**CMS:**
![image](https://user-images.githubusercontent.com/26731347/57053795-d7b61380-6cc2-11e9-813f-6308c5cbcccf.png)
(note URL in CMS)

**Comment by chrismplus** [comment]
Am I right in saying that if pull request #255 is implemented this issue will be resolved?

**Comment by arnasziedavicius** [comment]
> Am I right in saying that if pull request #255 is implemented this issue will be resolved? Other tasks such as the blue effect bug and the menu collapse seem to be behaving as expected in the new iteration of Staging...

@chrismplus, you are correct, it should get resolved anew once #255 is implemented. Will keep an eye on things here and confirm once it's on staging.

**Comment by arnasziedavicius** [comment]
Now resolved on staging.
